.cirrus.yml

```diff
@@ -96,14 +96,11 @@ linux_task_template: &LINUX_TASK_TEMPLATE
       
 tests_task:
   matrix:
-    - name: "${CIRRUS_OS} tests: iris=repo-main python=3.7"
-      env:
-        PY_VER: 3.7
-        IRIS_SOURCE: "source"
-    - name: "${CIRRUS_OS} tests: iris=repo-main python=3.8"
-      env:
-        PY_VER: 3.8
-        IRIS_SOURCE: "source"
+    # TODO: iris-grib does not currently work with iris-main
```

Comment:
I like that you reference #251 in the comment-out section below on doctests (lines 118-128)
Could we do something similar here?
It may mean making an issue to note the problem -- I think #277 also refers